models/clm/tutorial/README.rst

```diff
@@ -0,0 +1,1567 @@
+
++-----------------------------+-------------------------------------------------------------+
+| Run Time Assimilation Case  | Description                                                 |
+| Settings                    |                                                             |
++=============================+=============================================================+
+| ``DATA_ASSIMILATION_LND``   | If ``.TRUE.`` data assimilation is enabled.  If ``.FALSE.`` |
+|                             | no data assimilation is performed, and simulation is        |
+|                             | performed as a normal CLM run.                              |
++-----------------------------+-------------------------------------------------------------+
+| ``DATA_ASSIMILATION_SCRIPT``| Location of script ``assimilate.csh`` that performs the     |
+|                             | assimilation.  Controls execution of DART scripts and passes|
+|                             | files between DART and CLM code.                            |
++-----------------------------+-------------------------------------------------------------+
+| ``STOP_OPTION``             | Unit of time that controls duration of each assimilation    |
+|                             | cycle (see ``STOP_N``)                                      |
++-----------------------------+-------------------------------------------------------------+
+| ``STOP_N``                  | Duration of each assimilation cycle in units of time        |
+|                             | defined by ``STOP_OPTION``                                  |
++-----------------------------+-------------------------------------------------------------+
+| ``DATA_ASSIMILATION_CYCLES``| The number of assimilation cycles performed for a single    |
+|                             | job submission                                              |
++-----------------------------+-------------------------------------------------------------+
+| ``RESUBMIT``                | The number of times each job submission is repeated as      |
+|                             | defined by ``STOP_OPTION``, ``STOP_N``, and                 |
+|                             | ``DATA_ASSIMILATION_CYCLES``                                |
++-----------------------------+-------------------------------------------------------------+
+| ``CONTINUE_RUN``            | If ``.FALSE.`` this is the startup of the assimilation run  |
+|                             | beginning from the ``refcase``.  If ``.TRUE.`` this is a    |
+|                             | continuation of a previous assimilation run.                |
++-----------------------------+-------------------------------------------------------------+ 
+
+ 
+Before submission review your ``input.nml`` within your case folder to confirm
+the settings reflect those from previous steps of the tutorial.  Next submit
+the assimilation run:
+
+::
+
+ > cd <caseroot>
+ > ./case.submit
+
```

Comment:
What if a run fails? Maybe it's good to talk about both CESM and DART staging scripts here.